.github/workflows/github-actions.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         java-version: 1.11
         
     - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
+      run: git update-index --chmod=+x gradlew
```

Comment:
this command is in your machine and does a commit, i.e. this line should not be here